GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -750,7 +750,22 @@ private void PasteTextFromClipboard()
 
         private void TextBox_KeyPress(object sender, KeyPressEventArgs e)
         {
-            _disableAutoCompleteTriggerOnTextUpdate = e.KeyChar.IsSeparator();
+            bool isSeparator = e.KeyChar.IsSeparator();
+            _disableAutoCompleteTriggerOnTextUpdate = isSeparator;
+
+            if (e.KeyChar == (char)Keys.Back)
+            {
+               // When a character is deleted...
+               if (TextBox.TextLength == 0
+                    || TextBox.Text[TextBox.SelectionStart - 1].IsSeparator())
```

Comment:
```suggestion
                    || (TextBox.SelectionStart > 0 && TextBox.Text[TextBox.SelectionStart - 1].IsSeparator()))
```

Reply:
Thanks. I though it was equivalent to 1st condition but in fact, no and so there was a bug. Fixed another way.